rules/jboss-eap/weblogic/XmlWebLogicRules.windup.xml

```diff
@@ -157,7 +157,7 @@
             </when>
             <perform>
                 <iteration>
-                    <classification title="WebLogic Web Application Descriptor" effort="3"/>
+                    <classification title="WebLogic Web application descriptor" effort="3"/>
```

Comment:
Why keep Web with Upper W if there are following application descriptor in smallcaps?